### PR TITLE
doc: add list of known public Copr instances

### PR DESCRIPTION
**Pull request by FrostyX**
SSIA

### CODE REVIEW

File: doc/user_documentation.rst [hunk 1]
```diff
@@ -32,6 +32,22 @@ How to enable copr repository?
...
+
+Public Copr instances
+---------------------
+
+Copr is a free software and anyone can maintain their own instance in
+case the Fedora Copr instance doesn't suit their needs. This is a list
+of known Copr instances:
+
+- https://copr.fedorainfracloud.org - Fedora Copr instance which is
+  also considered to be the "main" Copr instance.
```
Comment (praiskup) [inline doc/user_documentation.rst:44]: I'd say *which is the oldest instance, and also considered to be the default Copr instance by many users and tools*

Comment (praiskup) [reply doc/user_documentation.rst:44]: I think it would be nice to cite a public issue tracker, or something alike ...

Comment (FrostyX) [reply doc/user_documentation.rst]: Good idea. So what about a table like this?

Screenshot_2023-06-01_00-18-42

File: doc/user_documentation.rst [hunk 2]
```diff
@@ -32,6 +32,42 @@ How to enable copr repository?
...
+
+   * - https://copr.stg.fedoraproject.org
+     - |
+       | Fedora Copr staging instance is useful for
+       | testing the upcoming changes.
+       | We periodically delete all its data.
+     -
+
+   * - https://eur.openeuler.openatom.cn
+     - OpenEuler Copr instance
```
Comment (TommyLike) [inline doc/user_documentation.rst:66]: sorry it's openEuler not OpenEuler 😄

Comment (FrostyX) [reply doc/user_documentation.rst]: Oh, you are right. Thank you.
Updated.

**Pull request merged**